Cargo.toml

```diff
@@ -20,6 +20,9 @@ rand = { version = "0.7.2", optional = true }
 hyper = "0.12.0"
 opentelemetry-jaeger = { path = "opentelemetry-jaeger", version = "0.1.0" }
 thrift = "0.13.0"
+futures = "0.1.25"
```

Comment:
Mind moving these deps so that they're only added for the example? (I can edit this PR or add a PR to your branch if you want.)

Reply:
please feel free to add commits on this one.

Reply:
Thanks!